src/xcode/ENA/ENA/Source/AppDelegate & Globals/CWAHibernationProvider.swift

```diff
@@ -20,7 +27,7 @@ class CWAHibernationProvider: RequiresAppDependencies {
 		#if RELEASE
 		return Date() >= hibernationStartDate
 		#else
-		return true // to.do Dev Menu setting, see https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14812
+		return secureStore.hibernationComparingDate >= hibernationStartDate
```

Comment:
My Suggestion hibernationComparisonDate or HibernationDate, please change it other places as well.

```suggestion
		return secureStore.hibernationComparisonDate >= hibernationStartDate
```

Reply:
adapted.